azure-devops/Create-PRDiff.ps1

```diff
@@ -0,0 +1,19 @@
+[CmdletBinding(PositionalBinding=$False)]

```

Comment:
Naming style: The other scripts in `azure-devops` are all `lower-case.ps1`.

Reply:
Clarification: I'm referring to the filename, not line 1. (Should have commented on an empty line.)